spring-cloud-gcp-docs/src/main/asciidoc/sql.adoc

```diff
@@ -47,7 +49,9 @@ public List<Map<String, Object>> listUsers() {
 }
 ----
 
-`JdbcTemplate` is configured based on the following properties:
+This starter is configured through
+https://docs.spring.io/spring-boot/docs/current/api/org/springframework/boot/autoconfigure/jdbc/DataSourceProperties.html[`DataSourceProperties`].
+There is also some configuration specific to Google Cloud SQL:
 
 |===
 | Property name | Description | Default value | Unused if specified property(ies)
```

Comment:
I think it's a bit confusing phrasing "Unused if specified property(ies)".
Maybe just list those properties in the table and say what overwrites what if specified.

Reply:
Let's fix this in another PR since it's unrelated here?